<launch>
  <include file = "$(find realsense2_camera)/launch/rs_camera.launch" />
  <arg name = "launch_prefix" default = "" />
  <arg name = "ns1" default = "apriltag_ros_continuous_node1"/>

  <rosparam command="load" file="$(find apriltag_ros)/config/settings.yaml" ns="$(arg ns1)" />
  <rosparam command="load" file="$(find apriltag_ros)/config/tags.yaml"     ns="$(arg ns1)" />
  
  <!-- <include file = "$(find ball_detect)/launch/sim_gps.launch" /> -->
  <node pkg="apriltag_ros" type="apriltag_ros_continuous_node" name="$(arg ns1)" clear_params="true" output="screen" launch-prefix="$(arg launch_prefix)">
    <remap from="image_rect"  to="/camera/color/image_raw" />
    <remap from="camera_info" to="/camera/color/camera_info" />
    <remap from="boat_pos"    to="/position"    />
    <param name="camera_frame" type="str" value="/rs" />
    <param name="publish_tag_detections_image" type="bool" value="true" />      <!-- default: false -->
  </node>

</launch>
